filament/src/details/View.cpp

```diff
@@ -56,7 +56,8 @@ static constexpr float PID_CONTROLLER_Ki = 0.002f;
 static constexpr float PID_CONTROLLER_Kd = 0.0f;
 
 FView::FView(FEngine& engine)
-        : mFroxelizer(engine),
+        : mEngine(engine),
```

Comment:
I have resisted all this time storing an `Engine` in `View`, I don't want to add that dependency (in fact, I wish I had been more diligent about this in other classes).
So instead, you can just store the "isStereoSupported" in a `bool:1`.